.github/workflows/reusable_package_mage.yaml

```diff
@@ -54,14 +54,18 @@ on:
         type: string
         description: "Destination directory in S3 bucket"
         default: "mage-unofficial"
+      large_runner:
+        type: string
+        description: "Use large runner?"
```

Comment:
Why would the user ever say "no" this this option? Explain the cost/benefit.

Reply:
We use github hosted runners for this workflow. There are two types of github runners, basic (let's call them) and large. Both are billed on a pay as you go basis. A fixed amount of basic runner minutes are included in memgraph's monthly subscription plan, large runner minutes aren't at all. So the cost here is :heavy_dollar_sign: . The reason for using large runners here is because the dev build takes up too much disk and the basic runners can't handle it.

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners
https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates

Reply:
So the default is false, which means unless the user overrides - it will always pick basic and fail the build. This doesn't seem logical to me.

Reply:
I understand your concern, let me clear things up.
The user will never be able to manually run the `reusable_package_mage.yaml` workflow.
It is only called by the `release_artifact.yaml` and `package_mage.yaml` workflows. 
I have put in safeguards for this with the following lines:
  - `large_runner: ${{ github.event.inputs.mage_build_target == 'dev' && 'true' || 'false' }}`
  - `large_runner: ${{ matrix.build_target == 'dev' && 'true' || 'false' }}`
Additionally, all builds except `dev` will pass on a basic runner.